tables/akk.tbl

```diff
@@ -0,0 +1,958 @@
+
+# liblouis: Akkadian
+#
+# ------------
+#-index-name: Akkadian uncontracted
+#-display-name: Akkadian uncontracted
+#
+#+language:akk
+#+type:literary
+#+contraction:no
+#+system:akk-oracc
+
+#-license: lgpl-2.1
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+#for more information on the Akkadian language, please go to:
+#https://oracc.museum.upenn.edu/dcclt/signlists/signlists/
+#The standard for Akkadian has been set by the academic community represented by ORACC. The braille code for Akkadian follows the standard set by ORACC. The braille code for Akkadian is represented in braille as the name for the sign in Akkadian. Thus, if the Akkadian sign is a "Lum," then the braille code for this sign would be lum.
```

Comment:
There are some things in this explanation that I find confusing.

> The standard for Akkadian [...]

Which standard are we talking about exactly? Not a braille standard, right?

> The braille code for Akkadian follows the standard set by ORACC. The braille code for Akkadian is represented in braille as the name for the sign in Akkadian. [...]

After having looked at some rules I think I understand, but a better example would help. There is no documentation for the braille code, except this one paragraph. So IMO it can be a bit more elaborate. Specifically I would change "the braille code for this sign would be lum" to "the braille representation of this sign would be dots 123-136-134, which is the English braille representation of the word "lum". I say English because this is what it seems to be: in the rule for 𒈕 (lu sheshig) the "sh" is mapped to dots 146, which is what the English do. (Perhaps also some other braille systems, but it's not the international standard.)

Reply:
This paragraph has been removed completely, while I did findit helpful. It only needed a bit of clarification.

Reply:
And e.g. the Ugaritic table also still has a similar paragraph.